src/test/java/lambda/part3/exercise/Exercise3.java

```diff
@@ -0,0 +1,108 @@
+package lambda.part3.exercise;
+
+import lambda.data.Employee;
+import lambda.data.JobHistoryEntry;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"unused", "ConstantConditions"})
+public class Exercise3 {
+
+    private static class LazyMapHelper<T, R> {
+
+        private List <R> source;
+
+        private LazyMapHelper(List<R> source){
+            this.source = source;
+        }
+
+        public static <T> LazyMapHelper<T, T> from(List<T> list) {
+            // TODO реализация
+            return new LazyMapHelper<>(list);
+        }
+
+        public List<R> force() {
+            // TODO реализация
+            throw new UnsupportedOperationException();
```

Comment:
You must implement this method as terminal operation.

Reply:
I just don't understand the difference between methods force() and getMapped().  In MapHelper we use getMapped(), in LazyCollectionHelper we use force(). But here we are suggested to use both. Why so?

Reply:
There is no difference. Use method that you want, but then remove another one.

At initial task there was no `getMapped()` method, only `force()`: https://github.com/elefus/epam-lab-java8-2018-06/pull/147/commits/3b6e6884d0506aa1db15194110fbda310ae858ea